src/main/java/thrift/model/transaction/Description.java

```diff
@@ -22,6 +22,8 @@ public Description(String description) {
 
     @Override
     public String toString() {
+        assert value != null;
```

Comment:
When `toString()` is called, you assume that `value` is not `null`? It could return `null` according to Java specs.

Reply:
What do you mean? If the `Description` object is created, the `value` cannot be null due to the way it is constructed.